libs/StateSync.Managed/src/NetworkManager.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.MixedReality.Sharing.StateSync
+{
+    public interface NetworkManager
+    {
+        NetworkConnection GetConnection(InternedBlobRef connectionString);
+
+        bool PollMessage(NetworkListener listener);
+    }
+
+    /// <summary>
+    /// Maintains the cache of C++ wrappers for the created channels.
+    /// </summary>
+    public class NetworkManagerPInvokeWrapper
+    {
+        private readonly NetworkManager manager_;
+
+        // A connection created on the C# side can be wrapped by a NetworkConnection object on C++ side,
```

Comment:
```suggestion
        // A connection created on the C# side can be wrapped by a PInvokeNetworkManager::Connection object on C++ side,
```
Reduces ambiguity.

Reply:
That's an implementation detail: an internal class of a class from anonymous namespace. Certainly has no reason to be mentioned here.